packages/web/components/cards/stake-learn-more.tsx

```diff
@@ -5,7 +5,7 @@ import { Step, Stepper, StepsIndicator } from "~/components/stepper";
 
 export const StakeLearnMore: React.FC<{}> = () => {
   return (
-    <GenericMainCard title={"Get started with staking"} width="45">
+    <GenericMainCard title={"Get started with staking"}>
```

Comment:
Thoughts on localizing this before we forget it?

Reply:
Yea I agree. I probably missed this in a previous story. @mattupham can you move this into the en.json file before merging?